aodntools/ncwriter/imos_template.py

```diff
@@ -3,8 +3,7 @@
 
 from netCDF4 import num2date
 
-from ncwriter import DatasetTemplate
-
+from aodntools.ncwriter import DatasetTemplate
```

Comment:
The internal package imports are all relative except this one. Maybe should be:

```python
from .template import DatasetTemplate
```

?

Reply:
:+1: Done